styles/simple.scss

```diff
@@ -1,18 +1,106 @@
 .n-newsletter-signup--simple {
-	display: block;
+	display: flex;
+	flex-direction: row;
+	flex-shrink: 0;
 	border: 0;
+	gap: 20px;
+	padding-top: 20px;
+	background-color: oColorsByName("white-80");
+	padding-left: 24px;
+	padding-bottom: 0;
+
+	@include oGridRespondTo($from: M) {
+		align-items: start;
+		padding-left: 44px;
+		padding-bottom: 20px;
+	}
+
+	.n-newsletter-signup__card_content {
+		display: flex;
+		flex-direction: column;
+		align-items: start;
+		padding-left: 0;
+
+		@include oGridRespondTo($from: M) {
+			flex-direction: row;
+			align-items: center;
+		}
+
+	}
+
+	.n-newsletter-signup__body {
+		@include oTypographySans($scale: 0);
+		color: oColorsByName('black-80');
+		display: inline-block;
+		width: fit-content;
+
+		@include oGridRespondTo($from: M) {
+			width: 70%;
+		}
+	}
+
+	.n-newsletter-signup__top-meta {
+		top: 0;
+		right: 0;
+	}
+
+
+	.n-newsletter-signup__flex-child-left {
+		position: relative;
+		text-align: center;
+		color: oColorsByName('claret');
+		flex: 0 0 100%;
+		max-width: 158px;
+		max-height: 89px;
```

Comment:
**Question:** These feel like very specific numbers, do you know why they are so specific?